go/enclave/rollupchain/rollup_chain.go

```diff
@@ -667,57 +668,19 @@ func (rc *RollupChain) ExecuteOffChainTransaction(encryptedParams common.Encrypt
 	return encryptedResult, nil
 }
 
-// Extracts and validates the relevant parameters in a Call request.
-func extractCallParams(decryptedParams []byte) (gethcommon.Address, gethcommon.Address, []byte, error) {
```

Comment:
This was moved across into the RPC utils file, and broken into three separate methods.